subprojects/language-groovy/src/testFixtures/groovy/org/gradle/testing/fixture/GroovyCoverage.groovy

```diff
@@ -61,14 +73,32 @@ class GroovyCoverage {
         }
     }
 
+    private static boolean isCurrentGroovyVersionStable() {
+        !GroovySystem.version.endsWith("-SNAPSHOT")
+    }
+
     private static List<String> versionsAbove(List<String> versionsToFilter, String threshold) {
```

Comment:
I wonder if `versionsAtLeast` and `versionsAtMost` would make it clearer that the boundaries are included.

Reply:
To me it doesn't, but I don't feel strongly. Do you?